diff --git a/3rdparty/3rdparty.pri b/3rdparty/3rdparty.pri
index bc7c8b44..98749384 100644
--- a/3rdparty/3rdparty.pri
+++ b/3rdparty/3rdparty.pri
@@ -25,13 +25,6 @@ if(win32*):exists( $$PWD/qredisclient/qredisclient.lib ) {
 #PyOtherSide
 include($$PWD/pyotherside.pri)
 
-# Google breakpad
-BREAKPADDIR = $$PWD/gbreakpad/src
-DEPENDPATH += $$BREAKPADDIR
-
-INCLUDEPATH += $$BREAKPADDIR/
-INCLUDEPATH += $$BREAKPADDIR/src
-
 #breakpad app need debug info inside binaries
 win32-msvc* {
     QMAKE_CXXFLAGS += /MP
@@ -49,19 +42,6 @@ win32* {
         QMAKE_LFLAGS_RELEASE=
     }
 
-    HEADERS += $$BREAKPADDIR/common/windows/string_utils-inl.h
-    HEADERS += $$BREAKPADDIR/common/windows/guid_string.h
-    HEADERS += $$BREAKPADDIR/client/windows/handler/exception_handler.h
-    HEADERS += $$BREAKPADDIR/client/windows/common/ipc_protocol.h
-    HEADERS += $$BREAKPADDIR/google_breakpad/common/minidump_format.h
-    HEADERS += $$BREAKPADDIR/google_breakpad/common/breakpad_types.h
-    HEADERS += $$BREAKPADDIR/client/windows/crash_generation/crash_generation_client.h
-    HEADERS += $$BREAKPADDIR/common/scoped_ptr.h
-    SOURCES += $$BREAKPADDIR/client/windows/handler/exception_handler.cc
-    SOURCES += $$BREAKPADDIR/common/windows/string_utils.cc
-    SOURCES += $$BREAKPADDIR/common/windows/guid_string.cc
-    SOURCES += $$BREAKPADDIR/client/windows/crash_generation/crash_generation_client.cc
-
     ZLIBDIR = $$PWD/zlib-msvc14-x64.1.2.11.7795/build/native
     INCLUDEPATH += $$ZLIBDIR/include
     LIBS += $$ZLIBDIR/lib_release/zlibstatic.lib
@@ -93,7 +73,7 @@ unix:!macx { # ubuntu & debian
         #QMAKE_LFLAGS = -Wl,-rpath=/home/user/Qt5.9.3/5.9.3/gcc_64/lib
     }
 
-    LIBS += $$BREAKPADDIR/client/linux/libbreakpad_client.a -lz
+    LIBS += /usr/lib64/libbreakpad_client.a -lz
 
     # Unix signal watcher
     defined(LINUX_SIGNALS, var) {
diff --git a/src/modules/crashhandler/crashhandler.cpp b/src/modules/crashhandler/crashhandler.cpp
index 818dac2d..c03d1dc2 100644
--- a/src/modules/crashhandler/crashhandler.cpp
+++ b/src/modules/crashhandler/crashhandler.cpp
@@ -7,7 +7,7 @@
 #include <QFileInfo>
 
 #if defined(Q_OS_LINUX)
-#include "client/linux/handler/exception_handler.h"
+#include <breakpad/client/linux/handler/exception_handler.h>
 #elif defined(Q_OS_WIN32)
 #include "client/windows/handler/exception_handler.h"
 #elif defined(Q_OS_MAC)
diff --git a/src/rdm.pro b/src/rdm.pro
index d681b307..65a2f1f5 100644
--- a/src/rdm.pro
+++ b/src/rdm.pro
@@ -62,6 +62,8 @@ THIRDPARTYDIR = $$PWD/../3rdparty/
 
 include($$THIRDPARTYDIR/3rdparty.pri)
 
+INCLUDEPATH += /usr/include/breakpad
+
 win32 {
     CONFIG += c++11
 
